libnavigation-copilot/src/main/java/com/mapbox/navigation/copilot/MapboxCopilotImpl.kt

```diff
@@ -160,7 +169,10 @@ internal class MapboxCopilotImpl(
      */
     fun push(historyEvent: HistoryEvent) {
         val eventType = historyEvent.snakeCaseEventName
-        val eventJson = toEventJson(historyEvent.eventDTO)
+        val eventJson = when (historyEvent) {
```

Comment:
Will this produce the same result?
Just below you pass `eventJson` to `addActiveGuidance`. It probably expects some specific format.

Reply:
Good point, let me check

Reply:
fixed, thank you!